encryptor/queryDataEncryptor_test.go

```diff
@@ -524,10 +523,20 @@ schemas:
 			DataCoder:         &PostgresqlDBDataCoder{},
 			dialect:           postgresql.NewPostgreSQLDialect(),
 		},
-		// 1. different case table identifiers, postgresql
-		// should match, lowercase config identifier == lowercase SQL identifier
+		// 1. should partially match, like #0 but DEFAULT_client_id is not quoted and is processed as lowercase
 		{
-			Query:             `UPDATE "lowercasetable" set "other_column"='%s', "specified_client_id"='%s', "zone_id"='%s', "default_client_id"='%s'`,
+			Query:             `UPDATE lowercasetable set other_column='%s', specified_client_id='%s', zone_id='%s', DEFAULT_client_id='%s'`,
```

Comment:
hm... it doesn't match but processed without errors... can you create task to warn users in cases when we matched table but didn't match column if we have table schema? we should warn or even prevent processing cases with misbehavior from the user side. Leave now as is, but we should know about it and at least warning. It is worth a separate task.

Reply:
Maybe it is also possible to configure AcraCensor to warn/error on non-existing columns to catch this case? Oh, AcraCensor may even need changes like in this PR as well

Reply:
no, it is not responsibility of AcraCensor. and it has rules case sensitive for now. maybe if we will meet some issues with that, we will change it in the future